#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_ds_reference.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dir seperator for POSIX

@DPATCH@

--- ndoc-1.3.1.orig/src/VisualStudio/VisualStudio.build	2004-07-08 08:35:26.000000000 +0200
+++ ndoc-1.3.1/src/VisualStudio/VisualStudio.build	2005-09-25 22:14:11.000000000 +0200
@@ -9,6 +9,7 @@
 			<references>
 				<include name="System.dll" />
 				<include name="System.Xml.dll" />
+				<include name="System.DirectoryServices.dll" />
 			</references>
 		</csc>
 	</target>
